variables.tf

```diff
@@ -70,6 +70,12 @@ variable "reuse_cloud_connections" {
   default     = false
 }
 
+variable "cloud_connection_name_prefix" {
+  description = "Optional prefix for cloud connection name. If null default cloud connection name will be <zone>-conn-1. Else it <prefix>-<zone>-conn-1"
```

Comment:
`If null default cloud connection name will be <zone>-conn-1` -> `If null or empty string, default cloud connection name will be <zone>-conn-1.`